js/ui/scheduler/ui.scheduler.resource_manager.js

```diff
@@ -493,7 +493,7 @@ export default class ResourceManager {
         const currentResourcesData = [];
 
         resourceData.forEach(
-            data => currentResourcesData.push(Object.assign({}, data))
+            data => currentResourcesData.push(extend({}, data))
```

Comment:
@AntonSermyazhko can we use `extend` here?

Reply:
Yes. we can.

Reply:
Ok. Thank you, Anton!

Reply:
You are welcome, Alexander! :)